substrate/primitives/core/Cargo.toml

```diff
@@ -57,7 +57,7 @@ sp-runtime-interface = { path = "../runtime-interface", default-features = false
 # bls crypto
 w3f-bls = { version = "0.1.3", default-features = false, optional = true}
 # bandersnatch crypto
-bandersnatch_vrfs = { git = "https://github.com/w3f/ring-vrf", rev = "f4fe253", default-features = false, optional = true }
```

Comment:
Restore `rev` as soon as the following is merged:
- https://github.com/w3f/ring-vrf/pull/66
- https://github.com/w3f/ring-vrf/pull/67